testspiders/spiders/followall.py

```diff
@@ -15,7 +16,7 @@ def __init__(self, **kw):
         if not url.startswith('http://') and not url.startswith('https://'):
             url = 'http://%s/' % url
         self.url = url
-        self.allowed_domains = [urlparse(url).hostname.lstrip('www.')]
+        self.allowed_domains = [re.sub(r'^www\.','',urlparse(url).hostname)]
```

Comment:
I don't know if you have a way to add more changes to a PR using only the website, but if you do, please add spaces after parameter colons as [PEP8 mandates](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)